pkgs/tools/typesetting/soupault/default.nix

```diff
@@ -1,6 +1,7 @@
 { lib
+, stdenv
 , fetchFromGitea
-, fetchpatch
+, darwin
```

Comment:
Why these two additions?

Reply:
For fixing the package on darwin. See https://github.com/NixOS/nixpkgs/pull/242103/commits/07f3398415d0737ab441815272735364568a9bf9.

Reply:
Those I saw, but they are in the file `pkgs/development/ocaml-modules/camomile/default.nix`. My question is about the file `pkgs/tools/typesetting/soupault/default.nix` where these two lines are added but don't seem to be used anywhere.

Reply:
Oh, my! I'll remove them.